libs/common-system/src/test/java/org/codice/ddf/configuration/AbsolutePathResolverTest.java

```diff
@@ -23,42 +23,62 @@
 
 public class AbsolutePathResolverTest {
 
+  private static final AbsolutePathResolver TEST_APR =
+      new AbsolutePathResolver(Paths.get("test", "path").toString());;
+
+  private static final String USER_DIRECTORY = System.getProperty("user.dir");
```

Comment:
I would create a temporary folder, otherwise you risk polluting `$HOME` and potentially failing on a future run of the test if an orphan file is left there.